src/build/env.py

```diff
@@ -92,7 +92,9 @@ def __enter__(self) -> IsolatedEnv:
 
         :return: The isolated build environment
         """
-        self._path = tempfile.mkdtemp(prefix='build-env-')
+        # ``realpath`` is used to prevent warning from being emitted that
+        # the venv location has moved on Windows.
```

Comment:
This is to vague (and potentially misleading because the location does not actually move; the warning is due to a _misidentification_ of the location.

I’d say something like _Call realpath so venv does not incorrectly identify equivalent path forms as different locations on Windows_ with a reference to the b.p.o. issue entry.

Reply:
Expanded the text and linked to the issue.

Reply:
Thanks for the feedback :)